app/models/event.rb

```diff
@@ -234,6 +279,15 @@ class Event < ApplicationRecord
 
   has_many :organizer_position_invites, dependent: :destroy
   has_many :organizer_positions, dependent: :destroy
+
+  def ancestor_organizer_positions
+    OrganizerPosition.where(event_id: ancestor_ids)
+  end
+
+  def access_organizer_positions
```

Comment:
```suggestion
  def organizer_positions_with_access
```
imo this is more like English